client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -325,6 +366,61 @@ export default {
       return null
     },
 
+    statementSubmitterField () {
+      const attr = this.localStatement.attributes
+      let submitterField = 'authorName'
+      // If submitter is an orga and name has a value
+      if (attr.submitName && !attr.isSubmittedByCitizen) {
+        submitterField = 'submitName'
+      }
+
+      return submitterField
+    },
+
+    statementSubmitterValue: {
+      get () {
+        const attr = this.localStatement.attributes
+        let submitterValue = attr[this.statementSubmitterField]
+        // If submitter has revoked the gdpr consent for this statement or statement has been anonymized
+        if (this.isSubmitterAnonymous()) {
+          submitterValue = Translator.trans('anonymized')
+        }
+
+        return submitterValue
+      },
+      set (value) {
+        this.localStatement.attributes[this.statementSubmitterField] = value
+      }
+    },
+
+    submitterHelpText () {
+      const attr = this.localStatement.attributes
+      let helpText = ''
+
+      if (attr.gdprConsent && attr.gdprConsent.consentRevoked) {
+        helpText = Translator.trans('personal.data.usage.revoked')
+        if (hasPermission('area_statement_anonymize') &&
+          attr.original.submitterAndAuthorMetaDataAnonymized) {
+          helpText = helpText + `<br><br>${Translator.trans('statement.anonymized.submitter.data')}`
+        }
+      } else if (hasPermission('area_statement_anonymize') &&
+        attr.original.submitterAndAuthorMetaDataAnonymized) {
+        helpText = Translator.trans('statement.anonymized.submitter.data')
+      }
+
+      return helpText
+    },
+
+    submitterRole () {
+      let submitterRole = Translator.trans('institution')
+      if (this.localStatement.attributes.isSubmittedByCitizen &&
+          this.localStatement.attributes.submitterRole !== 'publicagency') {
+        submitterRole = Translator.trans('role.citizen')
+      }
+
+      return submitterRole
```

Comment:
nitpick: Here I would also store the condition in a variable, e.g. `isSubmittedByCitizen` (does that cover it?), and then you can simply do:

`return isSubmittedByCitizen ? Translator.trans('role.citizen') : Translator.trans('institution')`

Reply:
Done with [7baf002](https://github.com/demos-europe/demosplan-core/pull/2899/commits/7baf002e651d5fd0f294864061510e4ad4e74e95)